README.md

```diff
@@ -1,10 +1,18 @@
 # Proxless
 
-> Reduce your kubernetes cost by making all your deployments serverless with Proxless
+> Reduce your kubernetes cost by making all your deployments **on-demand** with Proxless
```

Comment:
comment - Sleep / Always On explaination.

Reply:
The famous confusion hehe.
Ok I'll make it clearer.